stock-tracker/src/store/reducer.js

```diff
@@ -3,12 +3,19 @@ export function reducer(state, action) {
         case 'ADD_RESPONSE':
             return {
                 ...state,
-                response: action.payload
+                response: action.payload,
+                loading: state.loading - 1
```

Comment:
Q: What was the rationale behind `loading` being a number?
Why not an object of keys to booleans, e.g:
```
loading = {
  news: true,
  chart: false,
  companyOverview: true,
}
```